backend/ccm/canvas_api/canvasapi_serializer.py

```diff
@@ -1,5 +1,11 @@
+from backend.ccm.models import Section
 from rest_framework import serializers
 
 class CourseSerializer(serializers.Serializer):
     # Define the fields you want to update. Adjust fields according to the Canvas API.
-    newName = serializers.CharField(max_length=255, required=True)
\ No newline at end of file
+    newName = serializers.CharField(max_length=255, required=True)
+
+class SectionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Section
+        fields = ['id', 'name', 'course_id', 'total_students', 'nonxlist_course_id']
```

Comment:
As far my understanding goes,  Here I don't think Section is Model in our Database. 
```
ModelSerializer in Django REST Framework (DRF) is a shortcut for creating serializers that map directly to Django models. 
```
https://www.django-rest-framework.org/api-guide/serializers/#modelserializer

What you have done here is what we did created a User serializer for `auth_user` table. HEre is the referance on this https://github.com/tl-its-umich-edu/canvas-course-manager-next/blob/develop/backend/ccm/serializer.py

Reply:
I don't think we need a serializer is needed for GET in this scenario. Since I only used serializer if request has some data to send like POST/PUT